DSCResources/MSFT_xDhcpServerOption/MSFT_xDhcpServerOption.psm1

```diff
@@ -259,7 +259,7 @@ function ValidateResourceProperties
                 $checkPropertyMessage = $($LocalizedData.CheckPropertyMessage) -f 'Dns server ip'
                 Write-Verbose -Message $checkPropertyMessage
                 
-                $dnsServerIP = ($dhcpOption | Where-Object Name -like 'DNS Servers').value
+                $dnsServerIP = ($DhcpOption | Where-Object OptionId -eq 6).Value
```

Comment:
@TravisEz13 Yes - where the option is set. Unlike the Name parameter, this is localised!